hack/generate.sh

```diff
@@ -106,7 +106,7 @@ ${KUBEVIRT_DIR}/tools/openapispec/openapispec --dump-api-spec-path ${KUBEVIRT_DI
 (
     cd ${KUBEVIRT_DIR}/docs
     ${KUBEVIRT_DIR}/tools/doc-generator/doc-generator
-    mv newmetrics.md ${KUBEVIRT_DIR}/docs/metrics.md
+    mv newmetrics.md metrics.md
```

Comment:
Why did you make this change?

Reply:
2 lines above we cd to this directory. So some of the code is relative path and some is full path - it is not consistant. Also, if `KUBEVIRT_DIR` does not start from root, then it will be failed, or moved to the wrong place.